roles/overpass-api/tasks/compile.yml

```diff
@@ -0,0 +1,68 @@
+- name: install packages
+  apt:
+    pkg: "{{ item }}"
+  with_items:
+    - autoconf
+    - automake1.11
+    - expat
+    - git
+    - g++
+    - libtool
+    - libexpat1-dev
+    - make
+    - zlib1g-dev
+    - bzip2
+    - wget
+    - liblz4-1
+    - liblz4-dev
+
+- name: download {{ overpass_release }} release of overpass-api
+  unarchive:
+    src: "https://github.com/drolbr/Overpass-API/archive/osm3s-{{ overpass_release }}.tar.gz"
+    dest: "{{ project_dir }}/"
+    validate_certs: false
+    remote_src: true
+  become: yes
+  become_user: "{{ user }}"
```

Comment:
I'm fine with downloading a tar.gz, but would it be a better idea to clone the git repository ?